website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/cloud-setting-up-bigquery-oauth.md

```diff
@@ -0,0 +1,46 @@
+---
+title: "Setting up BigQuery OAuth"
+---
+
+:::info Enterprise Feature
+
+This guide describes a feature of the dbt Cloud Enterprise plan. If you’re interested in learning more about an Enterprise plan, contact us at sales@getdbt.com.
+
+:::
+
+dbt Cloud supports [OAuth authentication](https://cloud.google.com/bigquery/docs/authentication) with BigQuery. When BigQuery OAuth is enabled, users can interact with the BigQuery warehouse as individual users, rather than leveraging a shared service level authentication. 
+
+:::info Some Pre-Work Required
+
+Before setting up a Client ID & Secret, you'll have to have your existing BigQuery Settings in order. We recommend using a Service Account JSON file, and have a walkthrough for that [here](https://docs.getdbt.com/tutorial/setting-up/#generate-bigquery-credentials)
```

Comment:
If you use a _relative url_ to a page in docs.getdbt.com, then docusaurus will do compile-time checks for broken links. Can you change this to
```
[here](tutorial/setting-up/#generate-bigquery-credentials)
```

I _think_ that should work!

Reply:
Good tip - I am still learning the details of docusaurus!

Reply:
ha - we caught this somewhere else - turns out docusaurus currently does not like relative links that end with a slash! I think we'll need to make this link:
```
[here](tutorial/setting-up#generate-bigquery-credentials)

```

ie. remove the trailing slash, at least until https://github.com/fishtown-analytics/docs.getdbt.com/pull/538 is merged :)